src/styleProps/marginProps.ts

```diff
@@ -1,4 +1,4 @@
-import {getSpacing} from '../utils';
+import getSpacing from '../utils/getSpacing';
 import {createStyleFunction} from '../utils/styleProps';
```

Comment:
These should both be importable from `'../utils'` as named imports, i.e. `import {getSpacing, createStyleFunction} from '../utils';`, might be a bit simpler. Or is there a reason other than consistency you've changed this?

Reply:
hum not sure why I changed these. changing them back.